.github/workflows/coverity_scan.yml

```diff
@@ -0,0 +1,60 @@
+name: coverity-scan
+on:
+  workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/coverity_scan.yml
+  schedule:
+    - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Install dependencies
+        run: pushd ../frontend/ && yarn install && popd
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Java Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: gradle-build-${{ hashFiles('*.gradle', 'gradle/dependency-locks/*') }}
+      - name: Download Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/java/linux64 --post-data "token=$TOKEN&project=CDCgov/prime-data-input-client" -O cov-analysis-linux64.tar.gz
+          mkdir cov-analysis-linux64
+          tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
+        env:
+          TOKEN: ${{ secrets.COVERITY_TOKEN }}
+      - name: Build with cov-build
+        run: |
+          export PATH=`pwd`/cov-analysis-linux64/bin:$PATH
```

Comment:
Are there multiple executables in there, or is it just `cov-build`?

Reply:
There are internal binaries as well. I pulled this from the ruby project. Seemed sane.